CHANGELOG/CHANGELOG-3.4.md

```diff
@@ -13,6 +13,9 @@ Previous change logs can be found at [CHANGELOG-3.3](https://github.com/etcd-io/
 ### Dependencies
 - Compile binaries using [go 1.19.10](https://github.com/etcd-io/etcd/pull/16038).
 
+### etcdutl v3
```

Comment:
3.4 doesn't have etcdutl.
```suggestion
### etcdctl v3
```

Reply:
minor comment: also usually we put the section `### Dependencies` at the end. So suggest to put this section to above it.

Reply:
Ah, too long since I updated last time.. Thank you very much! I will update changelog for 3.5 and 3.6 once this is merged.

Reply:
I think we can update all changelog in this PR. But it isn't a big deal.

Reply:
Done.